### PR TITLE
GOB: Small Improvements

### PR DESCRIPTION
**Pull request by BJNFNE**

### CODE REVIEW

File: engines/gob/detection/tables_playtoons.h [hunk 1]
```diff
@@ -337,7 +337,7 @@
 	kFeatures640x480,
 	"intro2.stk", 0, 0
 },
-{ // Supplied by indy4fan in bug report #13100. Orig title: "Der Mandarin-Prinz"
```
Comment (tag2015) [inline engines/gob/detection/tables_playtoons.h:340]: That's the nickname used in the bugreport

Comment (BJNFNE) [reply engines/gob/detection/tables_playtoons.h:340]: I know that the username on Discord is Indy4-Fan. And that is mostly used by the user.

Comment (tag2015) [reply engines/gob/detection/tables_playtoons.h:340]: Besides the fact that the usefulness of these changes is debatable, usernames can be changed. If in the future you decide to use a different username, are you going to change all the occurrences in the detection tables?

**Comment by bluegr** [comment]
Applied the second commit (add missing games) as 2c1f15efb8cc0a50cc32a195fb320e6a77431ed5

I agree with @tag2015 that the other commit (change to nickname who supplied X version) provides no value whatsoever

Thanks for your work! Closing this

**Pull request closed without merging**